src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -102,6 +103,27 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit {
   @Output()
   public isSelectedChange = new EventEmitter<boolean>();
 
+  public set childFocusIndex(value: number) {
+    const focusableChildren = this.adapterService.getFocusableChildren(
+      this.itemRef.nativeElement,
+      {
+        ignoreTabIndex: true
+      }
+    );
+    if (value !== this._childFocusIndex) {
+      this._childFocusIndex = value;
+      if (focusableChildren.length > 0 && value !== undefined) {
+        focusableChildren[value].focus();
+      } else {
+        this.itemRef.nativeElement.focus();
```

Comment:
Is there any way we could bury all of the DOM manipulations in the adapter service (here, and elsewhere)?

Reply:
OK, I moved the `focus` events to the adapter. Is that what you were referring to?